internal/pkg/build/sources/conveyorPacker_oci.go

```diff
@@ -26,6 +26,7 @@ import (
 	oci "github.com/containers/image/oci/layout"
 	"github.com/containers/image/signature"
 	"github.com/containers/image/types"
+	ocitypes "github.com/containers/image/types"
```

Comment:
Is there a reason to import this again under the `ocitypes` name instead of just using `types` as is being done elsewhere?